### PR TITLE
refactor!: migrated from cra to vite

### PR DESCRIPTION
**Pull request by Jaryt**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,72 +2,49 @@
...
+  "scripts": {
+    "dev": "vite",
+    "build": "tsc && vite build",
+    "preview": "vite preview"
+  },
   "dependencies": {
-    "@circleci/circleci-config-sdk": "v0.9.0-alpha.16",
-    "@craco/craco": "^6.3.0",
-    "@monaco-editor/react": "^4.3.1",
+    "@circleci/circleci-config-parser": "file:../circleci-circleci-config-parser-0.0.0-development.tgz",
```
Comment (KyleTryon) [inline package.json:12]: Still using a locally packaged version of the parser. I know the real one isn't published yet, but maybe this should wait for that.

File: src/components/atoms/form/ExecutorProperty.tsx [hunk 2]
```diff
@@ -0,0 +1,85 @@
+import { orb, reusable } from '@circleci/circleci-config-sdk';
```
Comment (KyleTryon) [inline src/components/atoms/form/ExecutorProperty.tsx:1]: Is this a new feature? not part of a refactor.

**Review by KyleTryon — CHANGES_REQUESTED** [review]
`delete public/vite.svg`

File: src/components/atoms/form/ExecutorProperty.tsx (hunk 2 above)
Comment (Jaryt) [reply src/components/atoms/form/ExecutorProperty.tsx:1]: This was already supported, just broke it out and just mixed it in with this refactor.

**Review by KyleTryon — APPROVED** [review]

**Pull request merged**